hosting_docs/source/reference/howto/wipe_persistent_data.rst

```diff
@@ -69,42 +74,25 @@ Rebuilding environment
 ----------------------
 
 
-#. 
-   Drop the first line public.yml. This step assumes the first line is
-   "wipe_environment_enabled: True", prepended in the steps above.
-
-   .. code-block::
-
-      $ tail -n +2 ~/environments/monolith/public.yml > public.yml.tmp
-      $ mv public.yml.tmp ~/environments/monolith/public.yml
+#. Remove the "wipe_environment_enabled: True" line in your `public.yml` file.
 
-#. 
-   Run Ansible playbook to recreate databases
+#. Run Ansible playbook to recreate databases
 
    .. code-block::
 
-      $ cchq monolith ap deploy_db.yml
+      $ cchq <env_name> ap deploy_db.yml
 
-#. 
-   Run management commands to create Kafka topics, create Postgres
+#. Run a code deploy to create Kafka topics, create Postgres
    tables, and Elasticsearch indices.
 
    .. code-block::
 
-      $ cchq monolith django-manage create_kafka_topics
-      $ cchq monolith django-manage preindex_everything
-      $ cchq monolith django-manage ptop_es_manage --flip_all_aliases
-      $ cchq monolith django-manage check_services
-      $ sudo -iu cchq  # Required to set CCHQ_IS_FRESH_INSTALL=1
-      (cchq) $ cd www/monolith/current
-      (cchq) $ source python_env/bin/activate
-      (cchq) $ env CCHQ_IS_FRESH_INSTALL=1 ./manage.py migrate_multi
-      (cchq) $ exit
+      $ cchq <env_name> deploy
+
 
-#. 
-   Recreate a superuser (where you substitute your address in place of
-   "you@your.domain").
+#. Recreate a superuser (where you substitute your address in place of
+   "you@your.domain"). (Optional)
 
    .. code-block::
 
-      $ cchq monolith django-manage make_superuser you@your.domain
+      $ cchq <env_name> django-manage make_superuser you@your.domain
```

Comment:
I have wondered about this step, especially when doing a domain import; should this superuser be created when the plan is to do a domain import? In the past I have created a superuser, but then when I imported the data (containing a superuser with same username), it failed because the user already existed. 

I think we should keep this step here in the spirit of the "rebuilding environment" document context, but if the reader comes to this document with the intent of importing data, this step might be confusing...any thoughts?